globaltracer.go

```diff
@@ -4,18 +4,18 @@ var (
 	globalTracer Tracer = NoopTracer{}
 )
 
-// InitGlobalTracer sets the [singleton] opentracing.Tracer returned by
+// SetGlobalTracer sets the [singleton] opentracing.Tracer returned by
 // GlobalTracer(). Those who use GlobalTracer (rather than directly manage an
-// opentracing.Tracer instance) should call InitGlobalTracer as early as
+// opentracing.Tracer instance) should call SetGlobalTracer as early as
 // possible in main(), prior to calling the `StartSpan` global func below.
-// Prior to calling `InitGlobalTracer`, any Spans started via the `StartSpan`
+// Prior to calling `SetGlobalTracer`, any Spans started via the `StartSpan`
 // (etc) globals are noops.
-func InitGlobalTracer(tracer Tracer) {
+func SetGlobalTracer(tracer Tracer) {
```

Comment:
Renaming this is a backwards breaking change. Suggest creating an alias instead, or incrementing the major version.